HIP/hip-646.md

```diff
@@ -0,0 +1,90 @@
+---
+hip: 646
+title: Fungible Token Metadata Field
+author: May Chan (@rocketmay)
+working-group: Ashe Oro (@Ashe-Oro)
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Review
+created: 2023-01-04
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/627
+updated: 2023-01-04
+---
+
+## Abstract
+
+This HIP proposes the addition of the metadata field to Fungible Tokens (FT), taking after the Non-Fungible Token (NFT) metadata field which was added in HIP-17.
```

Comment:
This may lead some readers to believe that HIP-17 gave metadata to non-fungible **token types**; instead of to individual NFT's. It also leaves it slightly unclear whether this HIP proposes to add `metadata` to both fungible and non-fungible token types; or only fungible types.

An extra sentence like,
```
While HIP-17 metadata is scoped to a single NFT, here we propose adding metadata scoped to an entire token type.
``` 
might help.

Reply:
Agreed, this is good clarification